### ISSUE TITLE
Landing Page Improvements - Page Restructure (Presentation Method)

### ISSUE THREAD

**Issue by jiraslama** [alan-turing-institute/AutisticaCitizenScience#347]
As a Fujitsu developer
I would like
A wireframe design and justification that showcases a revised landing page covering the feedback. 
Present less information, focusing on the get involved and configure sections being simplified further and different presentation techniques used to show the sections content. This will consist of the simplification of current three sections, representing of section content in accordions and the merging of the three sections content into one section.
The aims and how it works section and login being highlighted. For the Login feature to be centered and for it to be placed before How it Works & Aims sections.
Altering the Login button title to the title ready to begin or proceed or enter ! (provide screens of each of these)
Addition of a illustration or image to the How it Works section to indicate how it actually works in a visual form.
Make it clearer that the user can enter experiences in the slogan

**Comment by katoss** [comment]
# Original user testing feedback

From https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/community-recommendations/user-tests/sprint-6-feedback/summary-sprint6-feedback.md

**Amount of information/Understanding the purpose of the platform**

- P d: “It doesn’t look particularly confusing, although I don’t know why there is so much information on the landing page.”
- A (1) d: “I like the detail and the layout of this section.”
- A (2) d: “For me, it is just too much information. I mean, I realize it is really simple, but I guess that orange login-button is like a life-preserver, I would go for that.”
- A (2) d: “I am sure the purpose of the website would be clear if I had the energy to read it. It is simple, but people get frustrated that it is not even more clear.”
- A (2) d: “Visually it is very clean, but it is not very clear.”
- A C d: “It is very distracting, because you tell me what the aims are, now I just want to login and upload. Why would I need to see all this straight up?”
- A D d: It is not clear that you can enter experiences on this site
- A D d: “I would go on this website to find out things about autism or to help with it, but I cannot see where I could search for something. Normally you can see a search function right away.”
- A (2) d: “This landing page is giving you an overview of the site, but really it is trying to make it clear that you need to log in to begin. I think one thing that could be made better here [...] is to change the label ‘login’ to ‘ready to begin’ or ‘proceed’ or something. [...] It is like when you are doing a questionnaire, like one of these surveys for your company, and it is going to be like 100 questions and you just click ‘proceed’. It is very linear thinking, it is like ‘Ok, now I am just going to the next page.’. It proceeds in a linear fashion. As opposed to being on this landing page, and thinking ‘Alright, there are 12 sections and I don’t know where I want to start. I guess it is up to me.’ Like, I don’t really want to choose my own adventure, I want you to tell me the order [...], because that is easier for me. When I am at home, organising my bookshelf, I get to choose how I am going to organise it. And there are a million ways to do it and that is my choice, and I can struggle with that and use my energy for that. But when I am coming to this site, I just want you to tell me where I need to go next. Like with this session: Just tell me what I need to do. I need to save my energy for after this session.”

**Way of proceeding**

- P d: “I would probably go to ‘login’ and register.”
- A D d: Next step he would take: click on ‘Login’ button
- A (2) d: Reads first the aims, then ‘How it works’
- A (2) d: “Then I would have to log in, which is a bit stressful, because I don’t always know what all my login details are.”
- P d: “The first thing I read was the slogan. [...] Then I looked at ‘aims’, read them, and I didn’t really pay attention to ‘How it works’, ‘Configure’ and ‘Get involved’, because it looks like technical information, and then I went basically to ‘Login’.”
- P d: “I spent some time reading the slogan, because I was trying to understand whether it summarises everything or why it is there.”

**‘How it works’ section**

- A C d: “I want to look only at ‘how it works’.”
- A C d: “‘Authorise project’, what does that mean? Is that the consent-thing?”
- A C d: “I would maybe put ‘authorise project’ in brackets and write ‘give consent’, because that describes in plain English what ‘authorise project’ means.”
- A (1) d: “I would use the ‘How it works’, definitely. And I would like to see an example of how you would write that experience. It could be anonymous, but it could have a picture of how it would look. That would be useful.”

**‘Configure’ and ‘Get involved’ sections**

- A C d: “The ‘get involved’, I would rather like that as a postscript, later, and the ‘configure’, again, more like a postscript.”
- A (1) d: “I like the ‘Configure’ and ‘Get involved’-parts. I think it is not overcrowded and it is not too long.”
- A C d: “I would probably try to ignore the ‘get involved’ bit, because you can get distracted by that.”
- A C d: “So I am not sure if I would put the ‘get involved’ and ‘configure’ together with the ‘how it works’, because that is for the people who are more technical. As someone who only wants to login and enter experiences, I would not want to see the ‘configure’ and ‘get involved’.”

**Interface/Design**

- A (1) d: “I like how it is broken down into smaller sections, with borders around each box. I like the fact that they are separate, and that the links are highlighted.”
- A D d: “I find it quite glary, because it is really white and really bright. And I am strained looking at it, it is a bit too bright for me. It makes me want to look away.”
- A (2) d: “The text in the ‘How it works’ box should be one font size bigger.”
- A (2) d: “There are different options to design it, I am just saying it is not the best it could be yet.”
- A (2) d: “I think one thing that could be made better here [...] is making that login more centered [...]”
- A C d: “The ‘how it works’ - it means, you have got to login to Open Humans, that is part of the login. That would be straight underneath ‘aims’, before the login-button.”
- A C d: “I am not sure personally that I like the icon for ‘how it works’, because it looks like a ‘settings’ icon.”

**→ There is a tendency towards the perception that the landing page contains too much information that is irrelevant for the general user. Mainly the ‘get involved’ and ‘configure’ parts are criticised. The aims, ‘how it works’ and the login should be highlighted and the ‘how it works’ possibly illustrated with an example or images.**

**Comment by jiraslama** [comment]
Design Justification 

Improvements to landing page - the new design has adopted an accordion presentation method which is static / open on the How it works section with illustration / image of the experience page with content highlighted. The other less relevant / more technical information is now accessible by expanding the section accordion's below. 

Feedback has been actioned around the reordering of key call to actions with the Login to begin button being placed above the aims. 

The page examples are now ready for review by the community.

Please view in 1200 adaptive view

Landing Page Extended with illustration
https://k6ej5a.axshare.com/#g=1&p=1-landing-page-presentation-tech-expanded-with-ill&view=b

Landing Page Extended with image experience page
https://k6ej5a.axshare.com/#g=1&p=1-landing-page-presentation-tech-large-with-image&view=b

Landing Page Highlighted Ref - Fixed highlighting in neon blue highlighter on core copy - other content hoover demo as other landing page example also

https://k6ej5a.axshare.com/#g=1&p=1-landing-page-highlighted-ref&view=b

**Comment by jiraslama** [comment]
1-Landing Page Extended with Illustration 
<img width="799" alt="1-Landing Page Extended with Illustration" src="https://user-images.githubusercontent.com/41741006/93066088-71405400-f671-11ea-83c9-9a81e347dde3.png">

2-Landing Page Extended with Image of experience page
<img width="801" alt="2" src="https://user-images.githubusercontent.com/41741006/93066116-7bfae900-f671-11ea-8397-f0bd88cb277a.png">

3-Landing Page with highlighting options shown on old landing page design -
ref to highlighted examples on two landing pages above 1/2
<img width="803" alt="3" src="https://user-images.githubusercontent.com/41741006/93066149-861ce780-f671-11ea-9222-f57baac62d6d.png">

**Comment by GeorgiaHCA** [comment]
Hiya Chris, 

Thanks so much for this! I like the layout of this! It looks nice and clean. I have a couple of points: 

* I think this needs rephrasing: "What you did you to is login to upload our experiences"
* I think the goals are great and useful having them there.
* Autistica has changed their branding - https://www.autistica.org.uk/ - for more details on this, you can contact Rebecca Sterry I believe.

**Comment by GeorgiaHCA** [comment]
There are 3 main goals of the platform: 

* Share autistic people's stories and adaptive techniques with others who have similar experiences
* Educate neurotypical people to better support their autistic friends, familiy, and colleagues
* Advise organisations how they can design and adapt spaces to imporive autistic people's lives. 

Maybe these could be more clearly worded? It would be good to have these in the aims.

**Comment by jiraslama** [comment]
Came across need to alter How it works copy and already altered.

**Comment by jiraslama** [comment]
Aims altered to feedback from Georgia, I have also added Open Humans definition to the aims to act as a introduction to what OH is and how it is used by the platform. 

https://k6ej5a.axshare.com/#g=1&p=1-landing-page-presentation-tech-expanded-access-o_2&view=b

**Comment by jiraslama** [comment]
Added Style Guide sourced from Emma a few weeks ago as attached. 

Aware Brand for site had changed a number of months ago.

Added link to landing page wireframe which considers the new brand in relation to the CSP for review.

https://k6ej5a.axshare.com/#g=1&p=1-landing-page-style-guide-redesign&view=b

[202002 AUT_GUIDELINES_V1.pdf](https://github.com/alan-turing-institute/AutisticaCitizenScience/files/5244349/202002.AUT_GUIDELINES_V1.pdf)

**Comment by GeorgiaHCA** [comment]
Thanks @jiraslama - this is looking great! One small thing to change - 'life's' should be 'lives' - thanks so much for incorporating the new Autistica branding and using the style guide.

**Comment by fjThomasStanley** [comment]
Backlog refinement meeting (22/09/2020) - @GeorgiaHCA to look into including the how it works image ( https://k6ej5a.axshare.com/#g=1&p=icons-accessibility-how-it-works&view=b) into the community review letter for review.

**Comment by GeorgiaHCA** [comment]
Let's add @jiraslama's redesign to the deployed prototype! Lots of improvements in this issue, including:
* rebranding with Autistica's new design
* minor errors of phrasing/typos corrected 
* introduction of icons
* clearer wording

**Comment by jiraslama** [comment]
Break issue down into smaller compartments
